From f7fecd20c11652ae4296f9e585437f437a8b729b Mon Sep 17 00:00:00 2001
From: Kai Sommerfeld <kai.sommerfeld@gmx.com>
Date: Tue, 1 May 2018 09:34:59 +0200
Subject: [PATCH] [guiinfo] Fix Playr.IsInternetStream
 (PLAYER_ISINTERNETSTREAM) to use CFileItem::GetDynPath instead of
 CFileItem::GetPath.

---
 xbmc/guilib/guiinfo/PlayerGUIInfo.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/xbmc/guilib/guiinfo/PlayerGUIInfo.cpp b/xbmc/guilib/guiinfo/PlayerGUIInfo.cpp
index 0ec14833306b..9c716f582455 100644
--- a/xbmc/guilib/guiinfo/PlayerGUIInfo.cpp
+++ b/xbmc/guilib/guiinfo/PlayerGUIInfo.cpp
@@ -516,7 +516,7 @@ bool CPlayerGUIInfo::GetBool(bool& value, const CGUIListItem *gitem, int context
     case PLAYER_ISINTERNETSTREAM:
       if (item)
       {
-        value = URIUtils::IsInternetStream(item->GetPath());
+        value = URIUtils::IsInternetStream(item->GetDynPath());
         return true;
       }
       break;
